src/main/java/de/samply/share/client/rest/Api.java

```diff
@@ -130,4 +177,18 @@ private boolean checkUser(String basicAuth) {
     return BCrypt.checkpw(values[1], user.getPasswordHash());
   }
 
+  private HashMap<String, Object> filter(MultivaluedMap multivaluedMap) {
```

Comment:
I think we talked about headers already. Is this a consented protocol in NNGM? Were is it defined?